camera/camera_manager.cc

```diff
@@ -62,7 +62,11 @@ std::optional<int> CameraManager::follow_v4l_path(const std::string& path) const
   const std::string v4l_path = fs::canonical(path, ec).string();
   const int index = v4l_path.back() - '0'; // note that this will break for >10 cameras
   if (ec) {
-    return std::nullopt;
+    if (getenv("LOG_PATH")) { // if LOG_PATH is set, we can assume that we're in playback mode and the video index won't be used
```

Comment:
JAZ!@!!!!!!

Reply:
Some thoughts:

* Ideally, all of the code paths would be the same, online and offline, so they can't accidentally diverge
* Most consumers of the calibration don't need to actually interact with the camera, and the camerabq never needs the calibration -- those could be two different access paths
* Apparently you can directly open captures via the `v4l` path, so you may be able to remove the camera number munging

Reply:
Got it, I'll take closer look when I'm back in town!